clang/test/Parser/pragma-unroll.cpp

```diff
@@ -124,3 +124,40 @@ void test(int *List, int Length) {
 
 #pragma unroll
 /* expected-error {{expected statement}} */ }
+
+using size_t = unsigned long long;
+
+template <bool Flag>
```

Comment:
This test isn't particularly good, since it is validating that it 'parses', but that isn't our concern.  Please extract this to an AST test, and perhaps a codegen test.

Reply:
Agree!

Reply:
I've keep test case in this file, because clang currently has this issue https://godbolt.org/z/Wf46Er3qb